dev-java/core-specs-alpha/core-specs-alpha-0.2.56.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit java-pkg-2 java-ant-2
+
+EGIT_REF="053d789"
+
+MY_PN=core.specs.alpha
```

Comment:
You can use `ver_rs` here too, something to consider for the next version bump.

Reply:
So core_specs_alpha you mean?

Reply:
No, there's a `ver_rs` function that you can use to manipulate the version separators. So you can simply use it to replace `-` with `.` in `${PN}`.